tests/integration/valkey-cli.tcl

```diff
@@ -833,6 +843,37 @@ if {!$::tls} { ;# fake_redis_node doesn't support TLS
         close_cli $fd
     }
 
+    test "valkey-cli make sure selected db survives connection drops" {    
+        set fd [open_cli]
+        
+        # Target database
+        set db 9
```

Comment:
We already have `$::dbnum` which depends on `$::singledb`. See the first lines in this file.